.github/workflows/npm-test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           - 18.x
           - '20.0'
           - 20.x
-          - 21.x
+          - 22.x
```

Comment:
Can you also add `22.0` as for the other versions? We use those to check if there are no issues with an older version of a major node release.

Reply:
22.1 is fine then as well. Just the minimal version for which everything works.